wikipedia-sentences/README.md

```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
 ```sh
 python app.py -t index
 ```
-If you see the following output, it means your data has been correctly indexed. 
+The relevant Jina code to index data given your Flow's YAML definition breaks down to
+```python
+with Flow().load_config('flows/index.yml'):
+    f.index_lines(filepath='data/toy-input.txt', read_mode='r', batch_size=16, num_docs=10)
+```
+The Flow will interpret each line in the txt file as one Document.
+You can limit the number of indexed Documents with the `num_docs`
+argument.
+If you see the following output, it means your data has been correctly indexed.
 ```sh
 Flow@5162[S]:flow is closed and all resources are released, current build level is 0
 ```
 
-We recommend you come back to this step later and index the full wikipedia dataset for better results. To index the [full dataset](https://www.kaggle.com/mikeortman/wikipedia-sentences) (almost 900 MB) follow these steps:
+#### What if new data arrives that needs to be indexed?
```

Comment:
Could we add this section as **[Optional] Step 5.** 

I see the logic of adding it in the indexing section, but my primary concern is getting the example running with minimum interference. Once it's running, and they get that 'woohoo' moment, we can build on top of it.

Reply:
Yep, that is a good idea. I moved it down (https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62)